README.md

```diff
@@ -1,16 +1,26 @@
-SimpleLogin - privacy-first email alias and Single Sign-On (SSO) Identity Provider
+SimpleLogin - One of the first privacy respecting Email Aliasing and Single Sign-On (SSO) Identity Service(s).
 ---
 
-https://simplelogin.io
+# Bio
+This repository contains the everything related to the website of SimpleLogin SAS.
 
-This repo contains the code behind the main website at https://www.simplelogin.io
+# Online Presence
+[Website](https://simplelogin.io)
+[Twitter](https://twitter.com/simple_login)
+[Reddit](https://www.reddit.com/r/Simplelogin/)
+[Producthunt](https://www.producthunt.com/products/simplelogin)
```

Comment:
I think we can remove Producthunt and Other part here

Reply:
Can be done.